app/colors/catppuccin.css

```diff
@@ -0,0 +1,200 @@
+    --suggestions-file: #a6adc8;
+    --notification-border: #11111b;
+    --notification-date: #a6adc8;
+    --notification-permission: #a6adc8;
+    --notification-dialog: var(--notification-permission);
+    --notification-error: #f38ba8;
+    --notification-warning: #f9e2af;
+    --notification-info: #89dceb;
+    --notification-success: #a6e3a1;
+    --url-hover-fg: #bac2de;
+    --url-hover-bg: #313244;
+    --screenshot-highlight: #f5c2e7;
+    --screenshot-highlight-background: #f5c2e7;
+    /* special pages */
+    --link-color: #cba6f7;
+    --link-underline: none;
+    --scrollbar-bg: #313244;
+    --scrollbar-thumb: #45475a;
+    --button-disabled: #45475a;
+    --code-fg: #f5e0dc;
+    --code-bg: #181825;
+    --code-command: #f2cdcd;
+    --placeholder-text: #a6adc8;
+    --special-page-element-bg: #313244;
+    --special-page-element-border: #313244;
+    --input-unfocused: #313244;
+    --input-focused: #585b70;
+    --download-progress-fg: #585b70;
+    --download-progress-bg: #313244;
+    --helppage-h1: #cdd6f4;
+    --helppage-h2: #cdd6f4;
+    --helppage-h3: #cdd6f4;
+    --helppage-countable: #f9e2af;
+    --helppage-range-compat: #f9e2af;
+    --helppage-nativetheme-fg-light: #5c5f77;
+    --helppage-nativetheme-bg-light: #eff1f5;
+    --helppage-nativetheme-fg-dark: #bac2de;
+    --helppage-nativetheme-bg-dark: #1e1e2e;
+    --history-current-page-highlight: var(--helppage-h1);
+    /* sourceviewer */
+    --syntax-fg: var(--fg);
+    --syntax-bg: var(--bg);
+    --syntax-keyword: #f38ba8;
+    --syntax-entity: #a6e3a1;
+    --syntax-constant: #94e2d5;
+    --syntax-string: #89dceb;
+    --syntax-variable: #eba0ac;
+    --syntax-comment: #a6adc8;
+    --syntax-entity-tag: #cba6f7;
+    --syntax-markup-heading: #a6e3a1;
+    --syntax-markup-list: #f5c2e7;
+    --syntax-markup-emphasis: #94e2d5;
+    --syntax-markup-addition-fg: #cba6f7;
+    --syntax-markup-addition-bg: #181825;
+    --syntax-markup-deletion-fg: #f2cdcd;
+    --syntax-markup-deletion-bg: #f38ba8;
+/*
+ * APP
+*/
+/* navbar & tabs */
+#tabs, #navbar {width: 100vw;display: flex;background: var(--bg);counter-reset: tab-counter 0;}
+#logo {-webkit-app-region: drag;height: 1.8em;width: 1.8em;margin: .1em;min-width: 1.8em;min-height: 1.8em; display: none;}
+#mode-container {display: flex;width: 4em;text-align: center;margin: 0;justify-content: center; align-items: center;font-size:0.7em; font-weight: normal;}
+#mode {text-transform: capitalize;font-size: 1.3em;font-weight: bold;display: flex;align-items: center; line-height: 1;display: none;}
+/* modes */
+[current-mode=command] #mode-container::before,
+[current-mode=insert] #mode-container::before,
+[current-mode=normal] #mode-container::before,
+[current-mode=follow] #mode-container::before,
+[current-mode=explore] #mode-container::before,
+[current-mode=search] #mode-container::before,
+[current-mode=pointer] #mode-container::before,
+[current-mode=visual] #mode-container::before {
+  font-size: 1.5em;
+}
```

Comment:
I think these can be reduced to a single `#mode-container::before` selector instead of targeting each mode explicitly. The `::before` already makes it more specific than any default styling, so it's not needed to list each mode.